Apply fix from liquorice theme

bebehei opened the pull request:
Remove own style definitions in global.css. These had been either
conflicting or not neccessary.

Applies fix from eliasson/liquorice#13

tsipinakis commented:
Some of these changes were made purposefully and this PR breaks them, specifically:
* Breaks link color (had been purposefully set to blue to be more visible)
* Removes the backdrop from code blocks which was also purposefully set to make code stand out a bit more.

bebehei commented:
> Some of these changes were made purposefully and this PR breaks them, specifically:

TBH, I estimated those changes as not so important that it justifies to write conflicting rules against the themes' CSS.

> Breaks link color (had been purposefully set to blue to be more visible)

The link color had been very ugly in TOC. I fixed this now differently. I also think blue is a bit to strong (but that's not the scope of this PR).

> Removes the backdrop from code blocks which was also purposefully set to make code stand out a bit more.

Removing the lines and using the default was much easier and less conflicting, but here you go: I changed it.

bebehei commented:
ping @tsipinakis

tsipinakis commented:
Ooops sorry totally forgot about this.

And now I remember I why I didn't update liquorice earlier - the latest version requires a hugo version higher than the one currently available in debian - will have to look into installing it manually or pulling from a newest version before I can review this.

bebehei commented:
> Edit: Also the code blocks look really weird now

You probably forgot to update the submodule!?

![screenshot-2017-11-26 faq dunst](https://user-images.githubusercontent.com/2073458/33239721-3882e712-d2a8-11e7-8f53-c6d2a1d3a5c9.png)

This codeblock style is faulty. During development, I've seen this, too, but it mostly had been a problem of the submodule.

> Edit2: The Home position seems to be from the updated hugo version

I'd advocate to remove the menu line in `_index.md`. The Home page is already linked in the dunst heading on the left.

tsipinakis commented:
> I'd advocate to remove the menu line in _index.md. The Home page is already linked in the dunst heading on the left.

I'd say keep it, it doesn't hurt to be there anyway and while in many places the header of the files leads to the homepage it's not exactly a standard or obvious.

bebehei commented:
ping @tsipinakis, see #6
